app/views/request/show.html.erb

```diff
@@ -64,7 +64,14 @@
                                show_owner_update_status_action: @show_owner_update_status_action,
                                show_other_user_update_status_action: @show_other_user_update_status_action,
                                last_response: @last_response } %>
-
+                               
+          <%= render partial: 'request/reply',
+          locals: { info_request: @info_request,
+                    track_thing: @track_thing,
+                    show_owner_update_status_action: @show_owner_update_status_action,
+                    show_other_user_update_status_action: @show_other_user_update_status_action,
+                    last_response: @last_response } %>
```

Comment:
The `request/reply` partial only requires the `info_request` and `last_response` local variables.

Also want to check the source code alignment of the `locals` hash.